com.unity.postprocessing/PostProcessing/Shaders/Builtins/ScreenSpaceReflections.hlsl

```diff
@@ -109,7 +108,9 @@ float Vignette(float2 uv)
 float3 GetViewSpacePosition(float2 uv)
 {
     float depth = _CameraDepthTexture.SampleLevel(sampler_CameraDepthTexture, UnityStereoTransformScreenSpaceTex(uv), 0).r;
-    float4 result = mul(_InverseProjectionMatrix, float4(2.0 * uv - 1.0, depth, 1.0));
+    float4 result = float4(float2(2.0 * uv - 1.0) * float2(unity_CameraInvProjection[0][0], unity_CameraInvProjection[1][1]),
+                           depth * unity_CameraInvProjection[2][2] + unity_CameraInvProjection[2][3],
+                           _ZBufferParams . z * depth + _ZBufferParams . w); // Use _ZBufferParams as it accounts for 0...1 depth value range
```

Comment:
Does the bug only happen on desktop OpenGL? What about Vulkan or GLES3.0 for iOS/Android? Aras made the observation that empirically OpenGL depth is in 0..1 range not -1..1.  Is this what you've seen in RenderDoc? See discussion: https://forum.unity.com/threads/_zbufferparams-values.39332

Need to be sure this change doesn't change behavior on mobile. If you lack the hardware, loop in QA to help verify.

Reply:
So far I've only observed the bug on desktop OpenGL. I will be sure to loop in QA to verify on mobile.

I believe this change is consistent with link you shared. The initial bug appears to be a result of this shader treating the OpenGL depth as -1..1 instead of 0..1. The OpenGL depth being 0..1 was what I observed in RenderDoc.

Reply:
Ok, sound good. Looping in @ryanhy-unity for XR/postfx QA.

Reply:
I took a look at OpenGLES3 and Vulkan on the only Android device I have for testing here, the Quest and SSR weren't showing either before or after these changes.  So, no regressions with these changes but it also is not working.